File: src/main/java/org/gtreimagined/gtlib/gui/slot/SlotCell.java
package org.gtreimagined.gtlib.gui.slot;

import org.gtreimagined.gtlib.capability.IGuiHandler;
import org.gtreimagined.gtlib.gui.SlotType;
import net.minecraftforge.items.IItemHandler;

public class SlotCell extends AbstractSlot<SlotCell> {

    public SlotCell(SlotType<SlotCell> type, IGuiHandler tile, IItemHandler stackHandler, int index) {
        super(type, tile, stackHandler, index);
    }

}

File: src/main/java/org/gtreimagined/gtlib/gui/slot/SlotCell.kt
package org.gtreimagined.gtlib.gui.slot

import net.minecraftforge.items.IItemHandler
import org.gtreimagined.gtlib.capability.IGuiHandler
import org.gtreimagined.gtlib.gui.SlotType

class SlotCell(type: SlotType<SlotCell>, tile: IGuiHandler, stackHandler: IItemHandler, index: Int) :
    AbstractSlot<SlotCell>(type, tile, stackHandler, index)
